docs/docs/getting-started/first-steps.md

```diff
@@ -71,6 +71,12 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
 
     Run `constellation config instance-types` to get the list of all supported options.
 
+    :::tip
```

Comment:
should this be moved between "Fill the values ..." and "By default, ..."?

Reply:
I don't have strong feelings for either - I have moved as suggested!